revdep/problems.md

```diff
@@ -87,6 +301,122 @@ Run `revdepcheck::cloud_details(, "DiagrammeR")` for more info
       Note: found 1 marked UTF-8 string
     ```
 
+# dosearch
+
+<details>
+
+* Version: 1.0.8
+* GitHub: NA
+* Source code: https://github.com/cran/dosearch
+* Date/Publication: 2021-08-19 16:40:02 UTC
+* Number of recursive dependencies: 18
+
+Run `revdepcheck::cloud_details(, "dosearch")` for more info
+
+</details>
+
+## Newly broken
+
+*   checking examples ... ERROR
+    ```
+    Running examples in ‘dosearch-Ex.R’ failed
+    The error most likely occurred in:
+    
+    > ### Name: dosearch
+    > ### Title: Identify a causal effect from arbitrary experiments and
+    > ###   observations
+    > ### Aliases: dosearch
+    > 
+    > ### ** Examples
+    > 
+    ...
+    +   g_igraph <- igraph::set.edge.attribute(g_igraph, "description", 3:4, "U")
+    +   dosearch(data2, query2, g_igraph)
+    + }
+    Warning: `graph.formula()` was deprecated in igraph 2.0.0.
+    ℹ Please use `graph_from_literal()` instead.
+    Warning: `set.edge.attribute()` was deprecated in igraph 2.0.0.
+    ℹ Please use `set_edge_attr()` instead.
+    Error in graph_split[[x]] : subscript out of bounds
+    Calls: dosearch -> get_derivation_dag -> lapply -> FUN
+    Execution halted
+    ```
+
+## In both
+
+*   checking C++ specification ... NOTE
+    ```
+      Specified C++11: please drop specification unless essential
+    ```
+
+*   checking installed package size ... NOTE
+    ```
+      installed size is  6.5Mb
+      sub-directories of 1Mb or more:
+        libs   6.2Mb
+    ```
+
+# fanovaGraph
+
+<details>
+
+* Version: 1.5
+* GitHub: NA
+* Source code: https://github.com/cran/fanovaGraph
+* Date/Publication: 2020-10-07 12:10:02 UTC
+* Number of recursive dependencies: 98
+
+Run `revdepcheck::cloud_details(, "fanovaGraph")` for more info
+
+</details>
+
+## Newly broken
+
+*   checking examples ... ERROR
+    ```
+    Running examples in ‘fanovaGraph-Ex.R’ failed
+    The error most likely occurred in:
+    
+    > ### Name: estimateGraph
+    > ### Title: FANOVA graph estimation.
+    > ### Aliases: estimateGraph print.graphlist
+    > 
+    > ### ** Examples
+    > 
+    > # Ishigami function, true analytical values: D12 = D23 = 0, D13 =~ 3.374
+    > q.arg = list(list(min=-pi, max=pi), list(min=-pi, max=pi), list(min=-pi, max=pi)) 
+    > estimateGraph(f.mat=ishigami.fun, d=3, q.arg=q.arg, n.tot=10000, method="LiuOwen")
+    Error in (function (edges, n = max(edges), directed = TRUE)  : 
+      unused arguments (3, FALSE)
```

Comment:
WAT

Reply:
again related to how we pass the ... :eyes:

Reply:
now I'm wondering whether there are uncaught occurrences of the same issue for other deprecated functions, I'll check.

Reply:
although, in this case, I'm starting to think the error is one usage error that existed before but wasn't caught.

The code is `graph(as.vector(t(E)), d , FALSE)` where d is 3 and `as.vector(t(E))` is `c(1, 3)`. In `make_graph()`, the ... are for "when the graph is given via a literal". Here the edges are numeric. So we're in https://github.com/igraph/rigraph/blob/b806ec209943846380015cb5ba3a7e605799d93c/R/make.R#L612-L627 where the ... are actually not used since the `old_graph()` function doesn't have more arguments.

Reply:
although, why did it used to work

Reply:
``` r
igraph::make_graph(c(1, 3), 3, FALSE)
#> IGRAPH 628f020 U--- 3 1 -- 
#> + edge from 628f020:
#> [1] 1--3
igraph::graph(c(1, 3), 3, FALSE)
#> Warning: `graph()` was deprecated in igraph 2.0.0.
#> ℹ Please use `make_graph()` instead.
#> This warning is displayed once every 8 hours.
#> Call `lifecycle::last_lifecycle_warnings()` to see where this warning was
#> generated.
#> Error in (function (edges, n = max(edges), directed = TRUE) : unused arguments (3, FALSE)
```

<sup>Created on 2024-01-08 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>